python/sbp/jit/parse_float.py

```diff
@@ -0,0 +1,37 @@
+import cffi
+
+ffi = cffi.FFI()
+ffi.cdef("""
+float get_f32(unsigned char a, unsigned char b, unsigned char c, unsigned char d);
+float get_f64(unsigned char a, unsigned char b, unsigned char c, unsigned char d,
+	      unsigned char e, unsigned char f, unsigned char g, unsigned char h);
+""")
+
+source = """
+float get_f32(unsigned char a, unsigned char b, unsigned char c, unsigned char d) {
```

Comment:
Can we differentiate  the return value between float32 and float64? Presenting with Python built-in float the precision info is lost.